### PR TITLE
add apim stg

### PR DESCRIPTION
**Pull request by thomast1906**
### Change description ###
add apim stg

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: environments/test/test.tfvars [hunk 1]
```diff
@@ -7,6 +7,7 @@ private_ip_address = "10.141.32.132"
 destinations       = ["10.141.15.250", "10.141.31.250"]
 vnet_rg            = "ss-test-network-rg"
 vnet_name          = "ss-test-vnet"
+hub                = "prod"
```
Comment (timja) [inline environments/test/test.tfvars:10]: wrong hub? this should be non prod if you're doing test

File: azure-pipelines.yaml [hunk 2]
```diff
@@ -70,6 +70,9 @@ parameters:
       - deployment: "apim"
         environment: "dev"
         dependsOn: "Precheck"
+      - deployment: "apim"
```
Comment (timja) [inline azure-pipelines.yaml:73]: you've updated the test file for the other one but this is staging?

Comment (thomast1906) [reply azure-pipelines.yaml:73]: my bad - clicked wrong folder, should be stg

Comment (thomast1906) [reply azure-pipelines.yaml:73]: added to `stg.tfvars` now

**Review by timja — APPROVED** [review]

**Pull request merged**